### PR TITLE
lesson01 and lesson02 files

### PR DESCRIPTION
**Pull request by jkrecek3**
my lesson01 and 02 files are ready for review.

### CODE REVIEW

File: students/jim_krecek/lesson02/fizzbuzz.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+	if i%3 == 0:
+		if i%5 == 0:
+			print("FizzBuzz")
+		else:
+			print("Fizz")
+	elif i%5 == 0:
+		print("Buzz")
+	else:
+		print(i)
+
```
Comment (natasha-aleksandrova) [inline students/jim_krecek/lesson02/fizzbuzz.py]: the indentation is wrong here - should be *exactly* 4 spaces (no tabs)

File: students/jim_krecek/lesson02/series.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+	Once generalized that way, sum_series(n, 0, 1) should be equivalent to fibonacci(n).
+	And sum_series(n, 2, 1) should be equivalent to lucas(n).
+	"""
+	if n == 0:
+		return x
+	elif n == 1:
+		return y
+	else:
+		return (sum_series(n-2,x,y) + sum_series(n-1,x,y))
```
Comment (natasha-aleksandrova) [inline students/jim_krecek/lesson02/series.py]: no need for parens with `return` statement as it isn't a function -> `return sum_series(n-2, x, y) + sum_series(n-1, x, y)`

File: students/jim_krecek/lesson02/grid.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+	minus = "- "
+	endline = "| "
+	space = "  "
+	top = (plus + minus * (size))*num + plus
+	line = (endline+ ((size)*space))*num + endline
+	i=0
+	m=0
+	print(top)
+	while m < num:
+		while i < (size):
```
Comment (natasha-aleksandrova) [inline students/jim_krecek/lesson02/grid.py]: no need to have parens around a variable -> `while i < size`

**Comment by jkrecek3** [comment]
@natasha-aleksandrova thanks for the feedback! is it required that I go back into my files and re-upload them with changes to Git? or are these merely suggestions to keep in mind for the future?

**Comment by natasha-aleksandrova** [comment]
great question. if you got full credit then you are not required to resubmit; python style violations only affect assignment grading and not exercise grading.

you can take this feedback and improve on your future code or you can correct these and create another PR to merge the fixes - totally up to you!

**Pull request merged**